lib/blacklight/request_builders.rb

```diff
@@ -51,6 +51,14 @@ def solr_search_params_logic= logic
       deprecation_deprecate :solr_search_params_logic=
     end
 
+    def search_builder_class
```

Comment:
@cbeer yes, I'm not saying you were taking that away. I was just saying that was the important part to me.